src/ttsim/main_target.py

```diff
@@ -113,13 +124,14 @@ class Targets(MainTargetABC):
 @dataclass(frozen=True)
 class Labels(MainTargetABC):
     column_targets: str = "labels__column_targets"
-    grouping_levels: str = "labels__grouping_levels"
+    input_columns: str = "labels__input_columns"
```

Comment:
Apologies for the micro-management, but this needs to go up one more line because the three types of targets have to be one block for my sanity...